pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -516,6 +516,23 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     axes = [self._get_input_axis_to_returnn(input, axis) for axis in axes]
     return {"class": "reduce", "mode": self.mode, "axes": axes, "from": self._get_input_layer_name(input)}
 
+  def make_output_tensor_from_returnn(self, inputs_flat: List[Tensor], layer: LayerBase) -> Tensor:
+    tensor = super(Reduce, self).make_output_tensor_from_returnn(inputs_flat, layer)
+    if self.mode in ["max", "min", "mean"]:
+      # handle special case where input to Reduce is FullStatic. Here, we might already know the output and in case it
+      # is a SizeValue, we can set it for the TensorEntry
+      naming = Naming.get_instance()
+      assert len(inputs_flat) == 1
+      input_entry = naming.tensors[inputs_flat[0]]
+      from .variable import FullStatic
+      if len(input_entry.output_from_calls) == 1:
+        mod = input_entry.output_from_calls[0].module.module
+        if isinstance(mod, FullStatic):
```

Comment:
You should not have any check for `FullStatic` here. `FullStatic` should be totally irrelevant in this function.